src/panel-app/apps.component.js

```diff
@@ -118,28 +162,61 @@ body.dark {
   color: #F8F8F2;
 }
 
-& .table {
-  width: 100%;
+& [role="table"] {
+  display: grid;
+  grid-row-gap: var(--table-spacing);
+  grid-template-columns: 1fr;
+  padding: var(--table-spacing);
+  width: max-content;
 }
 
-& .table td, .table th {
-  padding: 2px 8px;
+& [role="columnheader"] {
+  color: var(--gray);
+  font-size: .9rem;
+  padding-left: .25rem;
 }
 
-& .table-header {
-  color: #66D9EF;
-  text-align: left;
+& [role="row"] {
+  align-items: center;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 2fr;
+  grid-column-gap: calc(var(--table-spacing) * 2);
 }
 
+& [role="row"] [role="cell"]:nth-child(1) { grid-area: 1 / 1 / 2 / 2; }
+& [role="row"] [role="cell"]:nth-child(2) { grid-area: 1 / 2 / 2 / 3; }
+& [role="row"] [role="cell"]:nth-child(3) { grid-area: 1 / 3 / 2 / 4; }
+& [role="row"] [role="cell"]:nth-child(4), 
+& [role="row"] [role="cell"].summary-action { grid-area: 1 / 4 / 2 / 5; }
```

Comment:
I need to refresh my grid knowledge... :D

Reply:
I cheated and used https://cssgrid-generator.netlify.com/